eclipse-theia/theia
# deps: perform `yarn upgrade`

#### What it does


The commit includes the upgraded `yarn.lock` file after performing a `yarn upgrade` to get a better representation of what downstream applications would use when building their own products for the first time. It allows the framework to confirm the version ranges it declares are valid and work.

_Dependencies Awaiting Review_:

- [x] [`nan@2.16.0`](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/2675)
- [x] [`glob@8.0.3`](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/2676)
- [x] [`vscode-languageserver-textdocument@1.0.5`](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/2677)
- [x] [`diff@5.1.0`](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/2678)

#### How to test


- confirm that CI passes successfully
- confirm through smoke tests that the application starts fine

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
